src/components/controls/controls.tsx

```diff
@@ -25,6 +25,17 @@ import {TreeOptionsInfo, MapOptionsInfo, AnimationOptionsInfo, PanelOptionsInfo,
   ExplodeTreeInfo, FrequencyInfo, MeasurementsOptionsInfo} from "./miscInfoText";
 import { AnnotatedHeader } from "./annotatedHeader";
 import MeasurementsOptions from "./measurementsOptions";
+import { useSelector } from "react-redux";
+
+// TODO: Move this to src/store.
+// https://react-redux.js.org/using-react-redux/usage-with-typescript#define-root-state-and-dispatch-types
+interface RootState {
```

Comment:
I haven't kept up to date with Redux's TS stuff (and our usage predates `@reduxjs/toolkit`) but this looks good. We might want to add `[propName: string]: any;` to indicate that there is more state than just controls, and then add it within controls to indicate that the controls state is much bigger than just these 3 properties.

Reply:
Good idea! Done in [force-push](https://github.com/nextstrain/auspice/compare/9714d4234bac0ea17c81ee1e00957b1583ab9c4a..c7dd987b9d0ceaca288839205689e4cb4fc61db5).

Reply:
Added some more context in [force-push](https://github.com/nextstrain/auspice/compare/82015f3afd43c409a602b9b113d78a999be90a1d..16e680b03e4f8373d9964fe695bf1012a57074eb).

Reply:
I looked into Redux Toolkit a bit more. Well, this is actually [bad practice](https://redux.js.org/usage/migrating-to-modern-redux#migrating-typescript-for-redux-logic) since Redux Toolkit is capable of automatically inferring types. I'm going to try some [migration](https://redux.js.org/usage/migrating-to-modern-redux) in a separate PR.

Reply:
Redux Toolkit PR: #1710

Reply:
Rebased this PR onto the above PR so there are less changes here.